### PR TITLE
Feature/local docs

### PR DESCRIPTION
**Pull request by TarVK**
This PR adds the docs for LM to this repository, while still leaving the responsibility for presenting these docs in the https://github.com/LaunchMenu/LaunchMenu.github.io repository

Before this PR is merged, the accompanying PR in the website repo should also be accepted: https://github.com/LaunchMenu/LaunchMenu.github.io/pull/9
And both PRs should be merged at the same time.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!--
+Thanks for your pull-request!
+Please:
+- Make sure you updated the CHANGELOG to include your change in accordance to the description in the readme: https://github.com/LaunchMenu/LaunchMenu/tree/development#changelog
+- Link to any Issues that your PR may fix/implement
+- Clearly describe the intent of your PR if no related Issue exists
```
Comment (sancarn) [inline .github/pull_request_template.md:6]: might be a good idea to also say "update the docs"

File: README.md [hunk 2]
```diff
@@ -94,3 +94,39 @@ To install all dependencies, simply run `yarn` in the root of the repository. Th
...
+#### Branches/merging
+
+This repo 2 permanent branches, and 4 more branch types:
+
+-   master: This branch is up to date with the latest release.
+-   development: This branch contains all finished features for the next release.
+-   feature/\[featureName]: Branches matching this pattern contain a unfinished new feature
+-   fix/\[bugName]: Branches matching this pattern contain a unfinished fix for a bug
+-   other/\[change]: Branches matching this pattern contain changes not fitting under `feature` or `fix`, such as code refactors
+-   release/\[versionNumber]: Branches matching this pattern contain commits for alpha, beta and final releases before merging them to master
```
Comment (sancarn) [inline README.md:108]: maybe a docs type? In case someone only changes the docs

File: .github/pull_request_template.md (hunk 1 above)
Comment (TarVK) [inline .github/pull_request_template.md:6]:
```suggestion
- Clearly describe the intent of your PR if no related Issue exists
- Update the relevant docs in the `docs/website` directory
```

File: README.md [hunk 3]
```diff
@@ -94,3 +94,39 @@ To install all dependencies, simply run `yarn` in the root of the repository. Th
 When you're working on either the core of LM or an applet, run `yarn dev` within `packages/core`. This will start the core of LM, launching the program.
 
 Then in the case of working on an applet, simply run `yarn dev` again, but now within the directory of the applet you're working on. HMR will automatically take care of reloading your applet in the running LM instance when any changes occur. Do notice however that you may need to exit and open any menus of your applet again to see the changes.
+
+#### Branches/merging
+
+This repo 2 permanent branches, and 4 more branch types:
```
Comment (TarVK) [inline README.md]:
```suggestion
This repo 2 permanent branches, and 5 more branch types:
```

File: README.md [hunk 4]
```diff
@@ -94,3 +94,39 @@ To install all dependencies, simply run `yarn` in the root of the repository. Th
...
+
+-   master: This branch is up to date with the latest release.
+-   development: This branch contains all finished features for the next release.
+-   feature/\[featureName]: Branches matching this pattern contain a unfinished new feature
+-   fix/\[bugName]: Branches matching this pattern contain a unfinished fix for a bug
+-   other/\[change]: Branches matching this pattern contain changes not fitting under `feature` or `fix`, such as code refactors
```
Comment (TarVK) [inline README.md]:
```suggestion
-   docs/\[change]: Branches matching this pattern contain documentation improvements
-   other/\[change]: Branches matching this pattern contain changes not fitting under `feature`, `fix` or `docs`, such as code refactors
```

**Pull request merged**